test/Microsoft.AspNet.Identity.InMemory.Test/InMemoryUserStore.cs

```diff
@@ -368,7 +368,7 @@ public Task<IList<TUser>> GetUsersInRoleAsync(string roleName, CancellationToken
         {
             if (String.IsNullOrEmpty(roleName))
             {
-                throw new ArgumentNullException("role");
+                throw new ArgumentNullException(nameof(roleName));
```

Comment:
Won't this change the result from "role" to "roleName"? Is that the right thing?